lib/ansible/utils/collection_loader.py

```diff
@@ -47,6 +49,12 @@ def __init__(self, config=None):
         elif self._n_configured_paths is None:
             self._n_configured_paths = []
 
+        # Append all ``ansible_collections`` dirs from sys.path to the end
+        for path in sys.path:
+            sys_installed_path = os.path.join(os.path.abspath(path), 'ansible_collections')
+            if os.path.isdir(sys_installed_path):
```

Comment:
How about avoiding duplicates?
```suggestion
            if (
            	os.path.isdir(sys_installed_path) and
            	sys_installed_path not in self._n_configured_paths
            ):
```